community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexPopulationJob.java

```diff
@@ -76,7 +77,8 @@ public IndexPopulationJob(IndexDescriptor descriptor, SchemaIndexProvider.Descri
                               FailedIndexProxyFactory failureDelegateFactory,
                               IndexPopulator populator, FlippableIndexProxy flipper,
                               IndexStoreView storeView, UpdateableSchemaState updateableSchemaState,
-                              Logging logging)
+                              Logging logging,
+            IndexingService.Monitor monitor)
```

Comment:
I have them setup using the JAR Mattias showed me but somehow sometimes it automatically fixes stuff for me and sometimes not!